examples/scraper/urls

```diff
@@ -0,0 +1,2 @@
+www.google.com
```

Comment:
In the original version I know you had several domains. Do we want that again for the final demo?